src/oop/oop1.py

```diff
@@ -17,3 +17,65 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+
+#base class
+#  [Vehicle]
+class Vehicle:
+    def __init__(self):
+      pass
+
+#  [Vehicle]
+#      |                
+#      v                
+# [GroundVehicle] 
+class GroundVehicle(Vehicle):
+    def __init__(self):
+      super().__init__()
+      pass
+
+
+#  [Vehicle]
+#      |                
+#      v                
+# [GroundVehicle]      
+#   |       
+#   v       
+# [Car]  
+class Car(GroundVehicle):
+    def __init__(self):
+      super().__init__()
+      pass
+
+
+#  [Vehicle]
+#      |                
+#      v                
+# [GroundVehicle]      
+#           |
+#           v
+#      [Motorcycle]
+class Motorcycle(GroundVehicle):
+    def __init__(self):
+      super().__init__()
+      pass
+
+#[Vehicle]->[FlightVehicle]
+class FlightVehicle(Vehicle):
+    def __init__(self):
+      super().__init__()
+      pass
+
+#[Vehicle]->[FlightVehicle]
+#                       |
+#                       v
+#                  [Airplane]
+class Airplane(FlightVehicle):
+    def __init__(self):
+      super().__init__()
+      pass
+
+#[Vehicle]->[FlightVehicle]->[Starship]
+class Starship(FlightVehicle):
+    def __init__(self):
+      super().__init__()
+      pass
```

Comment:
Good work here. As we spoke about we could have simply put pass inside the bodies of the classes here.